portfolio/src/main/java/com/google/sps/servlets/NewServlet.java

```diff
@@ -0,0 +1,29 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
+@WebServlet("/mySeverlet")
+public class NewServlet extends HttpServlet {
+    
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String[] tips;
+    tips = new String[3];
+    tips[0] = "take a deep breath";
+    tips[1] = "be grateful";
+    tips[2] = "read a book";
```

Comment:
Since it's a hard-coded array, can we inline the initialization so that we don't need to keep track of the size every time we change the array's content?
```
String[] tips = new String[] {
    "take a deep breath", 
    "be grateful", 
    "read a book"
};
```

Reply:
Thank you for all the feedback! I am going back to finish the things I missed from week 2!